samples/Client/DaprClient/Readme.md

```diff
@@ -24,9 +24,28 @@ Deleted State!
 Executing transaction - save state and delete state
 Executed State Transaction!
 State not found in store
+Published deposit event!
 Done
  ```
 
+If you want to invoke RoutingService via HTTP or GrpcService via gRPC, you should run [RoutingService](..\..\AspNetCore\RoutingSample) or [GrpcService](..\..\AspNetCore\GrpcServiceSample) sample firstly, then run this command:
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useRouting true
+```
+
+or
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useGrpcsample true
+```
+
+or pass these 2 options to invoke RoutingService via HTTP and GrpcService via gRPC:
+
+``` sh
+ dapr run --app-id gRPC_Client -- dotnet run --useRouting true --useGrpcsample true
+```
+
 ## Invoking Services
```

Comment:
The command in the section below needs to be updated as well to reflect the config.GetValue changes:-
## Handling RpcException
Run the controller sample as follows from samples/AspNetCore/ControllerSample directory:-	Run the controller sample as follows from samples/AspNetCore/ControllerSample directory:-
```	```
dapr run --app-id controller --app-port 5000 dotnet run	dapr run --app-id controller --app-port 5000 dotnet run
```	```
Run the client sample as follows from samples/Client/DaprClient directory. The "rpc-exception" argument invokes a route on the server side that causes it to throw an RpcException:-	Run the client sample as follows from samples/Client/DaprClient directory. The "rpc-exception" argument invokes a route on the server side that causes it to throw an RpcException:-
```	```
dapr run --app-id gRPC_Client dotnet run rpc-exception	dapr run --app-id gRPC_Client dotnet run rpc-exception

Reply:
I fixed the usage of the rpc-exception argument